src/test/java/tr/com/srdc/cda2fhir/testutil/generator/ClinicalDocumentMetadataGenerator.java

```diff
@@ -145,4 +150,13 @@ public INT genVersion(CDAFactories factories, String versionNum) {
 		return version;
 	}
 
+	public static void setStructuredBody(CDAFactories factories, ClinicalDocument document,
+			List<Component3> components) {
```

Comment:
Same comment as before (generators instead of fields). But not very important at this point.